seminaire-saint-sulpice-ecole-theologie.csl

```diff
@@ -505,15 +506,21 @@
     <label plural="never" suffix=" " variable="page" form="short"/>
     <text variable="locator" form="short"/>
   </macro>
-  <citation et-al-min="4" et-al-use-first="3" et-al-subsequent-min="3" et-al-subsequent-use-first="1">
-    <layout suffix="." delimiter="&#160;; ">
+  <macro name="number-citation">
+    <label plural="never" prefix="n°" suffix=" " variable="number" form="short"/>
+    <text variable="locator" form="short"/>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="3" et-al-subsequent-min="3" et-al-subsequent-use-first="1"
+      near-note-distance="1">
```

Comment:
Why the near-note-distance? If you're not using near-note as a condition, this doesn't do anything